### DISCUSSION TITLE
Is it possible that the options of a field are related to a previous selection?

### DISCUSSION THREAD

**Discussion by adrchen** [directus/directus#5181] [Archive]
Lets say I have a Collection called Countries, and within are two fields:
- Country
- State

For the example, these are the entries

ID: 1 Country: United States - State: Florida
ID: 2 Country: United States - State: California
ID: 3 Country: Canada - State: Ontario
ID: 4 Country Canada - State: Alberta

Then I have a second Collection called Soccer Games, and from there
- Team name
- Country (M2O from Countries Collection)
- State **

So, would it be possible that if I select first in Country United States, and the options in State would only be Florida and California?

**Comment by benhaynes** [comment]
Hey! What you're looking for is Conditional Fields, which can be tracked here: https://github.com/directus/directus/issues/3151

**Reply by adrchen** [reply]
Thanks for the reply, @benhaynes ! Just read and it's more like showing and hiding fields based on previous selections; the question is more about the answers that can be pulled off another record (not sure if I'm confusing things).

If I choose as country United States, then the field that has State would give me the option to choose either Florida or California (?)

**Reply by benhaynes** [reply]
The way that we're structuring the implementation will allow changing not only the visibility, but also interface options. For things like drop-downs, the interface choices are included within those options. 👍

**Comment by fitu86** [comment]
Maybe this will work for you, I use it to search for vehicles. Year, Maker, Model, (same like Country, State, City).

Years
![image](https://user-images.githubusercontent.com/16139127/116007798-1db6c100-a5d7-11eb-8b58-22fc768a6224.png)

Makers
![image](https://user-images.githubusercontent.com/16139127/116007822-332beb00-a5d7-11eb-8a7e-ca4aee9ad26d.png)

Models
![image](https://user-images.githubusercontent.com/16139127/116007780-011a8900-a5d7-11eb-8824-f0a33b378703.png)

Then you create the collection that relates the previous ones.

![image](https://user-images.githubusercontent.com/16139127/116007956-e399ef00-a5d7-11eb-88ac-27b667ac890a.png)
